src/main/java/net/openhft/chronicle/testframework/internal/process/InternalJavaProcessBuilder.java

```diff
@@ -16,37 +18,55 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-public final class InternalProcessRunner {
+public final class InternalJavaProcessBuilder implements JavaProcessBuilder {
```

Comment:
I deleted `InternalProcessRunner` because it's an internal class I'm guessing we don't need to deprecate it. For some reason Github has decided it and `InternalJavaProcessBuilder` are the same file.

Reply:
Gr8